react/README.md

```diff
@@ -373,7 +377,7 @@ This style guide is mostly based on the standards that are currently prevalent i
 
   - Always define explicit defaultProps for all non-required props.
 
-  > Why? propTypes are a form of documentation, and providing defaultProps means the reader of your code doesn’t have to assume as much. In addition, it can mean that your code can omit certain type checks.
+  > Why? propTypes are a form of documentation, and providing defaultProps means the reader of your code doesn't have to assume as much. In addition, it can mean that your code can omit certain type checks.
```

Comment:
The `'` is currently much more often used than `’` across the Airbnb JavaScript guides. So if we change everything in the React guide to `’`, we should also update the other guides accordingly.

One argument for not converting everything to `’`: It's hard for developers (thus, the people who are writing those guides) to consistently use `’` because it's so common and much easier to write `'` in a code editor. You can see this by comparing how often `'` is used over `’`.

Though it would be nice to use `’` everywhere, I suppose it's simply not feasible and developers are going to keep adding `'` to the documentation. (We could think about some tool that lints our README files…)

I've created a second PR based on this one where I'm replacing all `'` with `’` within the React guide, not the other guides: #1901. (I can create a separate PR for the other guides in case we decide to use `’` everywhere.)

Reply:
The main guide already uses `’` almost everywhere, modulo a few places which I'll take care of.